core-rust/state-manager/src/mempool/pending_transaction_result_cache.rs

```diff
@@ -282,6 +282,24 @@ pub enum RetryFrom {
     Whenever,
 }
 
+#[derive(Debug, Clone)]
+pub struct DynamicValidatedTransaction {
```

Comment:
_(note from author)_

There is also the naming option of `PendingExecutedTransaction`.